<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detail</class>
 <widget class="QWidget" name="Detail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Album</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>148</y>
     <width>378</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="AuthorLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="YearLabel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="GenreLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
